CHANGELOG.adoc

```diff
@@ -16,6 +16,11 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
 === Added
 
 - Define GraphTemplate and Graph API to support the Graph database
+- Added a DefaultGroupEntityMetadata in the `jnosql-mapping-reflection` module
+
+=== Removed
+
+-  Removed unnecessary CDI Extension (EntityMetadataExtension.java)
```

Comment:
Just to make sure on this one.
You did not remove it, but move it to the reflection package once this extension makes sense when it reads by reflection, right?

Reply:
No, but after your recommendation I'll keep the `EntityMetadataExtension` for CDI full environments.